docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -0,0 +1,971 @@
+
+After identifying or considering a region as lost, you should ensure that it doesn't reconnect, as this will hinder a successful recovery during failover and failback execution. In case this is just temporary, Zeebe can survive a region loss but will stop processing due the loss in quorum and ultimately fill up the persistent disk before running out of volume resulting in the loss of data. <!-- Manu had some good advice on this -->
+
+The **failover** procedure aims to temporarily restore operations by redeploying Camunda 8 within the same region to resume workflow engine functionality. During this period, Zeebe is unable to export or process new data until it achieves quorum and the configured Elasticsearch endpoints for the exporters become accessible, which is the outcome of the failover procedure.
+
+The **failback** procedure involves completely restoring the failed region, thereby restoring your dual-region setup to its full functionality.
+
+The following procedures are building on top of the work done in the [AWS setup guide](./../../platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md#deploy-camunda-8-to-the-clusters) about deploying Camunda 8 to a dual-region cluster. We assume you have your own copy of the [c8-multi-region](https://github.com/camunda/c8-multi-region) repository and previously done changes in the `camunda-values.yml`.
+
+Please ensure to have followed the points [environment prerequisites](./../../platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md#environment-prerequisites) and [deploy Camunda 8 to the clusters](./../../platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md#deploy-camunda-8-to-the-clusters) to have the required base to build upon.
+
+### Failover
+
+<Tabs queryString="failover">
+  <TabItem value="step1" label="Step 1" default>
+
+#### Ensure Network Disconnection
+
+<StateContainer
+current={<Three viewBox="140 40 680 500" />}
+desired={<Four viewBox="140 40 680 500" />}
+/>
+
+<div>
+
+#### Current
+
+The current state is that one of the regions is lost. This will result in Zeebe being unable to process anything new due to the loss in quorum, nor can it export data to Elasticsearch since one of the instances is unreachable. Neither would it export to the local region since exporters are
+invoked sequentially.
+
+#### Desired
+
+For the failover procedure, we need to ensure that the lost region does not accidentally reconnect. You should be sure it is lost, and if so, look into measures to prevent it from reconnecting by for example utilizing the suggested solution below to isolate your active environment.
+
+#### How to get there
+
+Potential approaches are the following:
+
+- [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+- Firewall rules to block the traffic from the lost region
```

Comment:
I changed it to the same as above.
```
Configure firewall rules to disable traffic flow between the clusters
```

You'll want to disable all traffic flow between the cluster, ingress / egress. Not sure how Zeebe behaves when it can find it but no response. Rather be safe with no traffic.